### PR TITLE
Translation ready

### LINKED ISSUE #15: Make plugin translation ready
Kinda makes sense given the purpose of the plugin itself!

### PR DESCRIPTION
**Pull request by polevaultweb**
Resolves #15
- [x] Remove pot grunt task and .pot file

### CODE REVIEW

**Comment by ocean90** [comment]
I don't think we need the pot task and the.pot file. This will be handled by translate.wordpress.org as soon as the plugin is in the w.org repo.

**Comment by polevaultweb** [comment]
@ocean90 good point! i've got my premium plugin hat still on!

**Comment by bradt** [comment]
Meh, this seems close to done, so promoting to 1.0.

**Comment by polevaultweb** [comment]
@bradt happy if I just remove the .pot generation stuff?

**Comment by bradt** [comment]
Sure

**Comment by polevaultweb** [comment]
@polevaultweb needs wordpress-plugin-dbi merged in [gardening]

**Comment by ocean90** [comment]
- [x] The PR should remove these lines too: https://github.com/deliciousbrains/GlotPress/blob/ee8353be2b124d4339c9cb1d06a03b081eb6d4ee/gp-settings.php#L63-L64

**Comment by polevaultweb** [comment]
This should be good to go now

File: gp-settings.php [hunk 1]
```diff
@@ -180,3 +177,6 @@ function gp_run_route() {
 	}
 }
 add_action( 'template_redirect', 'gp_run_route' );
+
+// Load the plugin's translated strings
+load_plugin_textdomain( 'glotpress', false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
```
Comment (ocean90) [inline gp-settings.php]: Should we use `GP_LANG_PATH` here? If not, then we should remove the constant, line 8 in this file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+npm-debug.log
+node_modules
```
Comment (ocean90) [inline .gitignore]: Looks like we don't need this file yet.

**Comment by ocean90** [comment]
Can we squash this PR into one commit please? (http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit)

File: gp-settings.php (hunk 1 above)
Comment (polevaultweb) [inline gp-settings.php]: Removing, as it isn't relative to WP_PLUGIN_DIR

File: .gitignore (hunk 2 above)
Comment (polevaultweb) [inline .gitignore]: :+1:

**Comment by polevaultweb** [comment]
@ocean90 just figuring out the rebase :)

**Comment by polevaultweb** [comment]
@ocean90 Phew! Squashed :)

**Comment by ocean90** [comment]
Thanks @polevaultweb. :+1:

**Pull request merged**